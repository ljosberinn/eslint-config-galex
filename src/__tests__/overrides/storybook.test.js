const { createStorybookOverride } = require('../../overrides/storybook');

describe('createStorybookOverride', () => {
  test('matches snapshot if storybook is false', () => {
    const project = {
      hasStorybook: false,
    };

    expect(createStorybookOverride(project)).toMatchSnapshot();
  });

  test('matches snapshot if storybook is present', () => {
    const project = {
      hasStorybook: true,
    };

    expect(createStorybookOverride(project)).toMatchSnapshot();
  });
});
