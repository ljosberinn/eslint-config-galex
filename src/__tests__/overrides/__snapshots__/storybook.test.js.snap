// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`createStorybookOverride matches snapshot if storybook is false 1`] = `null`;

exports[`createStorybookOverride matches snapshot if storybook is present 1`] = `
Object {
  "files": Array [
    "*.stories.*",
  ],
  "overrideType": "eslint-config-galex/storybook",
  "rules": Object {
    "import/no-anonymous-default-export": "off",
    "import/no-default-export": "off",
    "react/function-component-definition": "off",
  },
}
`;
